sass/_components.scss

```diff
@@ -326,6 +333,87 @@
     }
 }
 
+// Pullout Block - List
+//
+// markup:
+// <div class="pullout-block pullout-block__list">
+//      <div class="pullout-block__content">
+//          <p class="pullout-block__content-heading">Think long, win big&hellip;</p>
+//          <p class="pullout-block__content-subheading">A good macro idea can be worth millions of dollars&hellip;</p>
+//          <ul>
+//              <li>In 1992, legendary traders George Soros and Stanley Druckenmiller made more than $1 billion by betting on a decline in the British Pound.</li>
+//              <li>In 2007 and 2008, hedge fund manager John Paulson personally made over $3 billion by betting on the U.S. mortgage market meltdown.</li>
+//              <li>In 2009, hedge fund manager David Tepper believed the worst of the 2008 financial crisis was over for the U.S. banks, taking a huge position in bank stocks and made over $5 billion as banks recovered.</li>
+//          </ul>
+//      </div>
+// </div>
+// 
+// Styleguide 5.8
+.pullout-block__list {
+    font-weight: 800;
+
+    @include mobile {
+        &:before {
+            content: none;
+        }
+    }
+
+    .pullout-block__content-subheading {
+        margin-bottom: $baseline;
+    }
+
+    ul {
+        margin-left: 0;
+        list-style: none;
+
+        li {
+            position: relative;
+            &:before {
+                font-family: AppleGothic, "Lucida Sans", Impact, Verdana, sans-serif;
+                content: '■';
+                color: $color-torch-red;
+                float: left;
+                width: 1em;
+                margin-top: 2px;
+                margin-left: -1.2rem;
+                font-size: 1rem;
+            }
+
+            @include mobile {
```

Comment:
Can you reverse this again so mobile is default. Cant quite work out whats going on here either - is it an arrow for the bullet on mobile and a square on everything else?

Reply:
There was no mobile design for this block so I just made something up. Square for desktop, arrow for mobile. Will make mobile first now